docs/resources/files.md

```diff
@@ -0,0 +1,108 @@
+---
+subcategory: "Storage"
+---
+# databricks_files Resource
+
+This resource allows uploading and downloading files upto 2GB in octet-stream.
+
+## Example Usage
+
+In order to manage a file on Unity Catalog Volumes with Terraform, you must specify the `source` attribute containing the full path to the file on the local filesystem.
```

Comment:
if it's only for UC Volumes, should we incorporate this into the resource name?

Reply:
now that I've thought about it, would make sense to call it `databricks_volume_file`

Reply:
@alexott @nkvuong Files for now only supports UC Volumes but is designed to be extended for other use cases as well so if we incorporate that in name then it will be a breaking change in future.

Reply:
if that's the case, then let's call it `databricks_file` (singular)

Reply:
Updated